src/lib/OpenApi.ts

```
@@ -159,9 +183,11 @@ export function OpenApi({
       return []
     }
 
-    return Object.values(getSpec().paths).reduce((tags: string[], path) => {
+    const paths = getSpec().paths as OpenAPIV3.PathsObject
+
+    return Object.values(paths).reduce((tags: string[], path) => {
       for (const verb of httpVerbs) {
-        if (path[verb]?.tags) {
+        if (path && path[verb]?.tags) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)